tests/test_lightblue.py

```diff
@@ -2379,18 +2714,85 @@ def test_images_with_modular_container_image(
             ["dummy-content-set-1", "dummy-content-set-2"], ["openssl-1.2.3-2.module+el8.0.0+3248+9d514f3b.src"], repositories)
```

Comment:
`openssl-1.2.3-2.module+el8.0.0+3248+9d514f3b.src` is not a valid NVR, it includes arch of `src`, it will cause problem when freshmaker parse this and result in unexpected result.